Work1/task1.py

```diff
@@ -0,0 +1,16 @@
+# дом.задание1
+
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+print('Введите трехзначное число: ')
+a = int (input())
```

Comment:
Незачем использовать функцию `print`. Сообщение можно написать внутри круглых скобок в функции `input`.
```suggestion
a = int (input('Введите трехзначное число: '))
```